partials/tabs/projects.html

```diff
@@ -5,8 +5,39 @@
   <br>
   <div class="main-content container">
     <div class="col m8 col offset-m2 s12">
-      <div class="input-field searchbar">
-        <input ng-model="searchText" placeholder="Search for a project" id="search" type="search" class="validate"> </div>
+      <div class="filter-projects-inputs">
+        <div class="input-field searchbar">
+          <input ng-model="searchText" placeholder="Search for a project" id="search" type="search" class="validate"> </div>
+        <div class="filter-btn apply-flex center-content">
+          <a id="filters" class="waves-effect waves-light btn-large" ng-click="toggleFiltersDisplay()">
+            <b>Filter Projects</b>
```

Comment:
The filters button can be clicked before ng is ready, and it redirects the user to https://github.com/coala/projects/

Reply:
Actually, here problem is not angular. It is because of that `Fork me on GitHub` ribbon. The ribbon transparent background is overlaying over that `div` that's we're getting redirected. If you check on current website, you will the same issue there too. This pblm can be solved either reducing the size of ribbon or removing it completely. But I've solved this pblm by replacing the `silders`  to `navbar`

Reply:
Raise a bug about it, if the problem still exists elsewhere in the page.

It doesnt need to be solved in this PR if it was a pre-existing problem.

Reply:
Solved now! As we now have a `navbar`. The ribbon 🎀 doesn’t exist no more which makes this issue no more valid 🙂